sdk/src/main/kotlin/io/rover/sdk/ui/blocks/poll/ViewTextPoll.kt

```diff
@@ -27,7 +27,7 @@ internal class ViewTextPoll(override val view: LinearLayout) : ViewTextPollInter
         )
     }
 
-    private lateinit var optionViews: List<TextOptionView>
+    private val optionViews = mutableListOf<TextOptionView>()
```

Comment:
Why not just set the option views as an immutable list?  I notice that you're doing extra work to allow for views to be added to this list from multiple places and yet you're just setting it in setupOptionViews().

Ditto for the image poll side of things.